Source/WebInspectorUI/UserInterface/Views/TimelineOverview.css

```diff
@@ -25,25 +25,25 @@
 
 .timeline-overview {
     --timeline-sidebar-width: 197px;
+    --timeline-header-height: 23px;
+
+    display: grid;
```

Comment:
do we also want to set `overscroll-behavior: none` to prevent the UI from rubber-banding?

Reply:
The container for `.timeline-overview` does not scroll. It always takes up as much space vertically as it needs.

Reply:
oh oops i meant to comment this about the parent `.timeline-recording.content-view`